training/cogvideox_text_to_video_lora.py

```diff
@@ -199,6 +198,26 @@ def log_validation(
     return videos
 
 
+class CollateFunction:
```

Comment:
Mind explaining the change?

Reply:
Inline function raised an error with pickle in windows, moving collate function to a class instances solves the issue.